### DISCUSSION TITLE
启动游戏自动打开日志，并且在日志出现异常

### DISCUSSION THREAD

**Discussion by Whatrulook** [Meloong-Git/PCL#1152] [提问] [answered]
乃乃的，我开整合包老是卡住，日志半天走不动，最后一条永远是一样的，不知道咋解决

**Comment by WForst-Breeze** [comment, marked as answer]
麻烦补充一下日志文件，如果可以的话整合包文件也上传一下，谢谢>w<

**Reply by Whatrulook** [reply]
Legend of Skyblock II-r1.0

**Reply by WForst-Breeze** [reply]
请提供问题出现时的日志，谢谢~

**Reply by Whatrulook** [reply]
> 请提供问题出现时的日志，谢谢~
[错误报告-2023-3-4_15.44.04.zip](https://github.com/Hex-Dragon/PCL2/files/10887776/-2023-3-4_15.44.04.zip)

**Reply by Whatrulook** [reply]
> ```
> [15:39:27] [Client thread/ERROR] [journeymap]: Couldn't locate icons for Default: E:\BaiduNetdiskDownload\mc\.minecraft\versions\Legend%20of%20skyblock\mods\journeymap-1.12.2-5.7.1.jar\assets\journeymap\icon\entity
> [15:39:27] [Client thread/INFO] [journeymap]: Added entity icons from journeymap:icon/entity. Success: false
> [15:39:27] [Client thread/ERROR] [journeymap]: Couldn't locate icons for flat: E:\BaiduNetdiskDownload\mc\.minecraft\versions\Legend%20of%20skyblock\mods\journeymap-1.12.2-5.7.1.jar\assets\journeymap\theme\flat
> ```
> 
> 看来是JourneyMap的问题（？） 前面报了`ic2.core.util.Config$ParseException`，但不是Error……

所以每次都卡CraftTweaker是什么原因
